code/__defines/items_clothing.dm

```diff
@@ -22,19 +22,23 @@
 #define SLOT_TIE        0x4000
 #define SLOT_HOLSTER	0x8000 //16th bit - higher than this will overflow
 
-#define ACCESSORY_SLOT_UTILITY  "Utility"
-#define ACCESSORY_SLOT_ARMBAND  "Armband"
-#define ACCESSORY_SLOT_RANK     "Rank"
-#define ACCESSORY_SLOT_DEPT		"Department"
-#define ACCESSORY_SLOT_DECOR    "Decor"
-#define ACCESSORY_SLOT_MEDAL    "Medal"
-#define ACCESSORY_SLOT_INSIGNIA "Insignia"
-#define ACCESSORY_SLOT_ARMOR_C  "Chest armor"
-#define ACCESSORY_SLOT_ARMOR_A  "Arm armor"
-#define ACCESSORY_SLOT_ARMOR_L  "Leg armor"
-#define ACCESSORY_SLOT_ARMOR_S  "Armor storage"
-#define ACCESSORY_SLOT_ARMOR_M  "Misc armor"
-#define ACCESSORY_SLOT_HELM_C	"Helmet cover"
+#define ACCESSORY_SLOT_UTILITY	0x1
+#define ACCESSORY_SLOT_WEAPON	0x2
+#define ACCESSORY_SLOT_ARMBAND  0x4
+#define ACCESSORY_SLOT_DECOR    0x8
+#define ACCESSORY_SLOT_MEDAL    0x20
+#define ACCESSORY_SLOT_TIE		0x40
+#define ACCESSORY_SLOT_INSIGNIA 0x80
+#define ACCESSORY_SLOT_OVER		0x100
+//Should these really be 'accessory' accessories
```

Comment:
I'd be open to other suggestions.

Reply:
It's fine for now imo, might as well not worry about it until we run out of flags 🤡

Reply:
I always worry about that sort of thing, is the issue.

Reply:
There's no other "good" way to do it, really. The other way is expensive.

Worst comes to worst, you override the 'can attach accessory' proc on armor and offload all the armor flags to some other 'armor slots' var.